flambe/field/text.py

```diff
@@ -272,7 +339,7 @@ def _build_embeddings(self, model: KeyedVectors) -> Tuple[odict, torch.Tensor]:
                 else:
                     self.unk_numericals.add(self.vocab[token])
 
-                    if self.unk_init_all:
+                    if unk_init_all:
```

Comment:
I would rename to avoid the outer scope naming collapse. It would become clearer it is definitely a local variable